README.md

```diff
@@ -110,7 +110,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
   * exception metadata,
   * thread metadata,
   * process metadata.
-* Supports iOS, macOS and tvOS platforms.
+* Supports iOS, macOS and tvOS platforms ([minimum supported platforms](https://github.com/backtrace-labs/plcrashreporter#prerequisites)).
```

Comment:
Thanks, @vlussenburg for your feedback. Minimum supported platforms are already documented in the first section: https://github.com/backtrace-labs/backtrace-cocoa#backtrace but repeating that information does not hurt.